trimesh/intersections.py

```diff
@@ -446,6 +447,9 @@ def slice_faces_plane(vertices,
         Normal vector of plane to intersect with mesh
     plane_origin :  (3,) float
         Point on plane to intersect with mesh
+    face_index : ((m,) int)
+        Indexes of mesh.faces which to slice. When no mask is provided, the
+        default is to slice all faces.
```

Comment:
Adding this parameter to the helper seems like the simplest way to accomplish this.